packages/utilities/src/initializeFocusRects.ts

```diff
@@ -2,8 +2,7 @@ import { getWindow } from './dom';
 import { KeyCodes } from './KeyCodes';
 import { KeyboardEvent } from '../../../common/temp/node_modules/@types/react';
 
-export const FabricClassName = 'ms-Fabric';
-export const IsFocusVisibleClassName = 'is-focusVisible';
+export const IsFocusVisibleClassName = 'ms-Fabric--isFocusVisible';
```

Comment:
Maybe I'm missing something, but will this break any of the SCSS that currently depends on this class?

Reply:
I will double check that. Good call.